problems/loops.js

```diff
@@ -143,16 +228,33 @@ function fizzBuzzPart1() {}
  * @returns {Array}
 
  */
-function fizzBuzzPart2() {}
-
+const fizzBuzzPart2 = () => {
+  let newArray = [];
+  for (let i = 1; i <= 100; i++) {
+    if (i % 3 === 0) {
+      newArray.push("Fizz");
+    } else if (i % 5 === 0) {
+      newArray.push("Buzz");
+    } else if (i % 3 === 0 && i % 5 === 0) {
+      newArray.push("FizBuzz");
+    } else {
+      newArray.push(i);
+    }
+  }
+};
 /**
  * Takes in range (inclusive) and returns the sum of all numbers in that range.
 
  * @param {number} min number
  * @param {number} max number
  * @returns {number} sum all all numbers from min to max
  */
-function rangeSum() {}
+const rangeSum = (min, max) => {
+  let sum = 0;
+  for (let i = min; i <= max; i++) {
+    return (sum += i);
```

Comment:
we want to return sum at the end, after the loop. returning in loop will end the loop